### PR TITLE
Addressing Ben's s2.3 nit

### LINKED ISSUE #18: rephrase s2.3 last para
Ben noted is his comments:

    2.3, last paragraph: The paragraph is a single,
    convoluted sentence that is hard to parse. (It's
    also a comma splice). Please consider breaking
    into multiple simpler sentences.

### PR DESCRIPTION
**Pull request by seanturner**
Closes #18.

### CODE REVIEW

**Review by fluffy — APPROVED** [review]

**Review by fluffy — APPROVED** [review]

**Review by martinthomson — APPROVED** [review]
This text is gross, but the changes are fine.
